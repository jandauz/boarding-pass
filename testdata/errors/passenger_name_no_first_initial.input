M1DESMARAISALUCAAAAAA/EABC123 YULFRAAC 0834 326J001A0025 100