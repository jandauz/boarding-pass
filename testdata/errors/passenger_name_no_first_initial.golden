bcbp: Invalid data format:
  boarding pass data:
  | "M1DESMARAISALUCAAAAAA/EABC123 YULFRAAC 0834 326J001A0025 100"
  |    ^ got "DESMARAISALUCAAAAAA/"
  |
  = reason: data for "Passenger Name" must be 20 characters with trailing whitespaces where the last name must be at most 18 characters followed by "/" and an alpha initial
